### PR TITLE
Add new fields for growth.

### PR DESCRIPTION
**Pull request by dnesbitt61**
Remove virtual destructors.

### CODE REVIEW

File: valhalla/baldr/directededge.h [hunk 1]
```diff
@@ -40,9 +31,9 @@ class DirectedEdge {
   DirectedEdge();
 
   /**
-   * Virtual destructor.
+   * Destructor.
    */
-  virtual ~DirectedEdge();
+  ~DirectedEdge();
```
Comment (kevinkreiser) [inline valhalla/baldr/directededge.h]: why even have a destructor for a trivial class

File: valhalla/baldr/nodeinfo.h [hunk 2]
```diff
@@ -27,9 +27,9 @@ class NodeInfo {
   NodeInfo();
 
   /**
-   * Virtual destructor.
+   * Destructor.
    */
-  virtual ~NodeInfo();
+  ~NodeInfo();
```
Comment (kevinkreiser) [inline valhalla/baldr/nodeinfo.h]: delete this :)

**Comment by kevinkreiser** [comment]
looks good

**Comment by dgearhart** [comment]
:+1:

**Comment by gknisely** [comment]
:ship:

**Pull request merged**